### PR TITLE
LoadFromFileCacheTtlMs is not honored (*after* first request)

### PR DESCRIPTION
**Pull request by oschaaf**
Fixes https://github.com/pagespeed/ngx_pagespeed/issues/1418

### CODE REVIEW

File: net/instaweb/rewriter/in_place_rewrite_context.cc [hunk 1]
```diff
@@ -402,8 +402,19 @@ void InPlaceRewriteContext::FixFetchFallbackHeaders(
       headers->Replace(HttpAttributes::kEtag, HTTPCache::FormatEtag(StrCat(
                                                   id(), "-", rewritten_hash_)));
     }
+
+    // Determine if we need to use the implicit cache ttl ms or the implicit
+    // load from file cache ttl ms.
+    int64 implicit_ttl = Options()->implicit_cache_ttl_ms();
```
Comment (jmarantz) [inline net/instaweb/rewriter/in_place_rewrite_context.cc]: implicit_ttl_ms (paranoid about units I am :)

**Review by jmarantz — COMMENTED** [review]
nice catch.  It shouldn't be too hard to add a unit test, I think.

File: net/instaweb/rewriter/in_place_rewrite_context.cc (hunk 1 above)
Comment (oschaaf) [reply net/instaweb/rewriter/in_place_rewrite_context.cc]: done

**Comment by oschaaf** [comment]
@jmarantz I changed an existing unit test, WDYT?

File: net/instaweb/rewriter/in_place_rewrite_context.cc [hunk 2]
```diff
@@ -454,6 +461,16 @@ void InPlaceRewriteContext::RemoveRedundantRelCanonicalHeader(
                   ResponseHeaders::RelCanonicalHeaderValue(url_));
 }
 
+bool InPlaceRewriteContext::IsLoadFromFileBased() {
```
Comment (oschaaf) [inline net/instaweb/rewriter/in_place_rewrite_context.cc:464]: @jmarantz added this, I anticipate we may need it later for landing https://github.com/pagespeed/mod_pagespeed/pull/1492

**Comment by 0x42h** [comment]
Hi @oschaaf. Does this mean it's in master already? I'm new here.

Strike that. I just read... 
```
Only those with write access to this repository can merge pull requests.
```

**Comment by oschaaf** [comment]
@0x42h It's not in master yet, this code change is currently under review. If everyone is happy with it it will get merged to master.

**Review by jmarantz — APPROVED** [review]

**Pull request merged**